### DISCUSSION TITLE
Demo URL

### DISCUSSION THREAD

**Discussion by lorenzo-moni** [saul/demofile#275] [Questions] [answered]
How can I get the URL to download the demo of a MM match by using the match `sharecode`?

**Comment by saul** [comment, marked as answer]
Yes - see my answer here: https://github.com/saul/demofile/discussions/270#discussioncomment-415046

**Reply by lorenzo-moni** [reply]
thank you but I was looking for a way of doing this in NodeJS instead of C++. Is this possible? excuse my ignorance..

**Reply by saul** [reply]
The functionality doesn't overlap with demo files so it's out of scope for this library I'm afraid.

If you're looking for a way to do it with nodejs, this library looks promising: https://github.com/joshuaferrara/node-csgo

**Reply by lorenzo-moni** [reply]
Ok thank you I'll give it a try
